cmd/glasskube/cmd/list.go

```diff
@@ -137,6 +137,7 @@ func printClusterPackageTable(packages []*list.PackageWithStatus) {
 	if listCmdOptions.ShowDescription {
 		header = append(header, "DESCRIPTION")
 	}
+	header = append(header, "MESSAGE")
```

Comment:
please move the status column in front of the message so they are next to each other.